src/Mutator/MutatorFactory.php

```diff
@@ -174,6 +174,7 @@ private static function createFromNames(array $mutatorNames): array
         foreach ($mutatorNames as $mutatorClass => $settings) {
             $mutatorConfig = new MutatorConfig($settings);
 
+            // TODO: only pass the mutator config if necessary
```

Comment:
Prehaps we should have an interface for configurable mutators (`ConfigurableMutator`)? And do some kind of side injection.

```php

interface ConfigurableMutator
{
    public function configure(MutatorConfig $config): void;
}

finalclass TrueValue implements ConfigurableMutator, Mutator
{
    private $settings = [];
    
    public function configure(MutatorConfig $config): void
    {
        $this->settings = $config->getMutatorSettings()
    }
}
```

Reply:
Could do: I'm still hesitating on how to do so because I would like more specific VOs than `MutatorConfig`, to remove the settings parsing (and validation) from the mutators more upstream